freqtrade/strategy/resolver.py

```diff
@@ -78,6 +82,14 @@ def _load_strategy(
             # Add extra strategy directory on top of search paths
             abs_paths.insert(0, extra_dir)
 
+        if validators.url(strategy_name):
```

Comment:
Please stick to [urlparse](https://docs.python.org/3/library/urllib.parse.html) and handle possible exceptions

Reply:
working on this, any reason why to stick to urlparse?